README.md

```diff
@@ -110,16 +110,13 @@ Tempest was built by a team of Stanford PhDs---[Peter Lofgren](@plofgren) (lead
       - nodeAttributes A list of name and type for each attribute in your csv file. Attributes type 
         may only be 'string', 'int' (32 bit), 'bigint' (64 bit), or 'boolean'. Enter the attributes
         in the same order as they appear in the node file.
+      - One of your nodeAttributes must be id with type string.  This id mut be unique across all nodes of this type, for example
```

Comment:
Ah, ignore above comment. How about the following:
"One of your nodeAttributes must be called 'id'. This nodeAttribute must have type string, and must be unique across all nodes of this type.